src/widgets/_toolbars.scss

```diff
@@ -15,6 +15,17 @@ actionbar {
     &.flat{
         @extend .inline-toolbar;
     }
+
+    &.title {
+        font-size: 11pt;
+        font-weight: bold;
+        padding: 0 12px;
+    }
+
+    &.subtitle {
+        font-size: 9pt;
```

Comment:
is this necessary since that's the default size?

Reply:
As said in the title comment, it's necessary since label size might change with the user prefence, and enforcing a size using `pt` ensures it scales properly.